source/localizable/tutorial/ember-data.md

```diff
@@ -102,5 +102,5 @@ When we call `findAll`, Ember Data will attempt to make a GET request to `/renta
 
 You can read more about Ember Data in the [Models section](../../models/).
 
-Since we're using Mirage in our development environment, Mirage will return the data we've provided.
-When we deploy our app to a production server, we will need to provide a backend for Ember Data to communicate with.
+Since we've already set up Ember Mirage in our development environment, Mirage will return the data we requested.
+When we deploy our app to a production server, we will likely want to provide a backend server for Ember Data to communicate with to store and retrieve persisted data from.
```

Comment:
The last sentence might be confusing because we are talking in this paragraph about using mirage vs a remote server.  Will have to think about how to elaborate on providing a backend.
"...we will likely want to replace Mirage with a remote server for Ember Data to communicate with for storing and retrieving persisted data.  A remote server will allow for data to be shared and updated across users."